mono/metadata/opcodes.h

```diff
@@ -20,21 +20,23 @@ MONO_BEGIN_DECLS
 
 #define MONO_CUSTOM_PREFIX 0xf0
 
+// FIXME duplication
```

Comment:
Push into a new separate header? mono-opcodenum.h? Or alter includes? Or ok to duplicate? Temporarily?